etc/seed/seed.yaml

```
@@ -12,22 +12,27 @@ input:
       let regions = ["us-east-1", "us-west-1", "us-east-2", "us-west-2"]
       let zoneSuffixes = ["a", "b", "c", "d"]
 
-      let region = $regions.index(random_int() % $regions.length())
-      let zone = "%s%s".format($region, $zoneSuffixes.index(random_int() % $zoneSuffixes.length()))
+      let subject = "customer-%d".format(random_int(seed: timestamp_unix_nano()) % $max_subjects)
+      let time = (now().ts_sub_iso8601("P3D").ts_unix() + random_int(min: 60, max: 60 * 60 * 24 * 3)).ts_format()
+
+      let method = $methods.index(random_int(seed: timestamp_unix_nano()) % $methods.length())
+      let path = $paths.index(random_int(seed: timestamp_unix_nano()) % $paths.length())
+      let region = $regions.index(random_int(seed: timestamp_unix_nano()) % $regions.length())
+      let zone = "%s%s".format($region, $zoneSuffixes.index(random_int(seed: timestamp_unix_nano()) % $zoneSuffixes.length()))
+      let duration = random_int(seed:timestamp_unix_nano()) % 1000
```

```suggestion
      let duration = random_int(seed: timestamp_unix_nano(), max: 1000)
```

The problem is only with calculated values.